### DISCUSSION TITLE
Skill Calculator: show total XP required to reach a level.

### DISCUSSION THREAD

**Discussion by calvi7** [runelite/runelite#17693] [Ideas]
The current skill calculator only shows your current experience and target experience. It would be nice if it also had one line that said
"Required experience: $xpDiff", with xpDiff simply being whatever the target xp minus the current xp is.

I would love to contribute this myself, but I'm afraid my Java knowledge is lacking.

Thanks.

**Comment by Nightfirecat** [comment]
This already exists as a tooltip if you hover the "Target Experience" input field. We understand this isn't necessarily the most discoverable location though, so we'd welcome a design which made it more visible without standing out too much.

**Reply by calvi7** [reply]
Oh, I actually didn't know this. I appreciate the info, and will be using it going forward. But it is true that it's not the most intuitive approach to this.
